src/store/offers-data/selectors.ts

```diff
@@ -0,0 +1,20 @@
+import { createSelector } from '@reduxjs/toolkit';
+
+import { TOffersData, TState } from '../../types/state';
+
+import { NameSpace } from '../../const';
+
+export const getOffers = createSelector(
+  (state: TState) => state[NameSpace.Offers],
+  (state: TOffersData) => state.offers
+);
```

Comment:
Здесь и далее. Это простой селектор который не мутирует данные. Мемоизация через `createSelector` здесь излишняя 
```suggestion
export const getOffers = createSelector(
  (state: TState) => state[NameSpace.Offers].offers
);
```